core/src/main/scala/org/apache/gearpump/cluster/embedded/LocalRuntimeEnvironemnt.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.gearpump.cluster.embedded
+
+import com.typesafe.config.Config
+import org.apache.gearpump.cluster.client.{ClientContext, RuntimeEnvironment}
+import org.apache.gearpump.cluster.embedded.LocalRuntimeEnvironemnt.LocalClientContext
+
+class LocalRuntimeEnvironemnt extends RuntimeEnvironment {
+  override def newClientContext(akkaConf: Config): ClientContext = {
+    new LocalClientContext(akkaConf)
+  }
+}
+
+object LocalRuntimeEnvironemnt {
+  class LocalClientContext private (cluster: EmbeddedCluster)
```

Comment:
`LocalRuntimeEnvironment` and `LocalClientContext` may not be the most proper name here. Remember we have a `local` cluster ?